From: Jesse Sung <jesse.sung@canonical.com>
Subject: Unregister the SAP dummy interface on exit.

This fixes error messages when the rfkill switch is toggled:

bluetoothd[5416]: sap/manager.c:sap_server_probe() path /org/bluez/5416/hci0
bluetoothd[5416]: sap-dummy interface org.bluez.SimAccessTest init failed on path /org/bluez/test
bluetoothd[5416]: Sap driver initialization failed.
bluetoothd[5416]: sap-server: Operation not permitted (1

--- sap/sap-dummy.c	2012-03-16 17:07:43.626644579 +0800
+++ sap/sap-dummy.c	2012-03-16 17:10:32.774636738 +0800
@@ -340,6 +340,7 @@
 
 void sap_exit(void)
 {
+	g_dbus_unregister_interface(connection, SAP_DUMMY_PATH, SAP_DUMMY_IFACE);
 	dbus_connection_unref(connection);
 	connection = NULL;
 }
